### ISSUE TITLE
Confirmation of services/add on termination doesn't work (Canadian)

### ISSUE THREAD

**Issue by EHCanadian** [ovh/manager#2887]
**Describe the bug**
Any confirmation link to delete/terminate/remove/cancel a service doesn't work while
under a Canadian variant of the ovh portal. The links sent you to end up with ovh error pages.

**To Reproduce**
1. Select a service/option you wish remove
2. Fallow the prompts on the web page
3. Check your email
4. Click the link and after login (my account has F2A enabled)

**Expected behavior**
Confirmation and execution of task

Email Title: "Request for closure of services for the account ms*-ovh"
Confirmation Link: https://ca.ovh.com/manager/#/billing/confirmTerminate?id=*&token=*
Cancel Request: https://www.ovh.com/ca/en/support/resiliation/?action=cancel&t=*&n=ms*-ovh

Cancel gives: One or more settings are missing or invalid
Confirm gives: Sorry, an error has occurred whilst loading the page.

This has been going on for some time. So its not new.

**Comment by antleblanc** [comment]
Hello @EHCanadian! Thanks for reporting us this issue.

An internal ticket also refer to this known issue so I leave the ticket reference here to ease the process. We will keep you posted as soon as a patch will be deployed.

**Internal ticket reference**: DTRSD-11814

**Comment by marie-j** [comment]
Hello @EHCanadian , 

Sorry about the response delay, a patch has been deployed, can you confirm it's working on your side ? 

Thanks

**Comment by EHCanadian** [comment]
I don't have any services at this time to be removed or attempt fate with to test on.
I had to contact customer support to remove services that I needed to be removed.

The ovh manager doesn't have anything that I can see that I can add without cost and require
a confirmation link(s) to be accessed for the removal. 

But when something does arise. I'll keep this ticket bookmarked for a response.

**Comment by antleblanc** [comment]
Hi @EHCanadian!

I will close this issue (patch has been deployed).
Please do not hesitate to re-open this Issue if you are still able to reproduce it.

Thank  you!

**Comment by EHCanadian** [comment]
Its back. Well I had a issue with it.

`OVH HOSTING INC. - https://www.ovh.com/
1801 McGill College ave. | Suite 800
Montréal (Québec) H3A 2N4
Canada






01/05/2021 10:34:18 GMT-0400

Dear Customer,

You have received this email because a request for closure of services was made on our website for the ID ms000000-ovh (*@*).
This request relates to the *.ip-00-00-000.net domain, the DEDICATED_OPTION service, and the BANDWIDTH sous-type de service.

Important: 
If you shut down this service it will be permanently deleted. OVH will delete its configuration as well as any potentially associated data (files, emails, databases, etc.).

If you made this request and you still want us to close the service,
simply click on the link below, or copy it in your browser:

https://ca.ovh.com/manager/#/billing/confirmTerminate?id=37088227&token=IjjMrN9ianUHGfvgV2yh8gfpNZt170KMFKev4zUA9I75Moqjyc0dPMLua8W9H04x

This request was made from this IP address: 127.0.0.1

If you did not make this request, or that you wish to delete it, simply
click on the link below, or copy it into your browser:

https://www.ovh.com/ca/en/support/resiliation/?action=cancel&t=IjjMrN9ianUHGfvgV2yh8gfpNZt170KMFKev4zUA9I75Moqjyc0dPMLua8W9H04x&n=ms000000-ovh

Warning: This link is only valid for 72 hours.
After which, you will need to make a new request.

We thank you for choosing OVH and we are ready to assist you as required.


OVH Support
Call us at: 1-855-OVH-LINE (684-5463)
24/7/365


[ref=1.f80d6c81]
`

Now the problem is the confirmation link gives a "An error has occurred cancelling the service. Got an invalid (or empty) URL" after the questionnaire. I had to use the cancel link and change cancel to confirm

I got the https://ca.api.ovh.com/console/#/dedicated/server/{serviceName}/option/{option}#DELETE from support.
